client-sqldelight-gradle-plugin/src/test/projects/happyPath/lib/build.gradle.kts

```diff
@@ -0,0 +1,38 @@
+import app.cash.backfila.client.sqldelight.plugin.SqlDelightRecordSource
+
+plugins {
+  id("app.cash.backfila.client.sqldelight")
+  id("app.cash.sqldelight")
+  kotlin("jvm")
+}
+
+sqldelight {
+  databases {
+    create("HockeyDataDatabase") {
+      packageName.set("app.cash.backfila.client.sqldelight.hockeydata")
+      dialect(libs.sqldelightMysqlDialect)
+      srcDirs.setFrom(listOf("resources/migrations"))
+      deriveSchemaFromMigrations.set(true)
+      migrationOutputDirectory.set(file("$buildDir/resources/main/migrations"))
+      verifyMigrations.set(true)
+    }
+  }
+}
+
+backfilaSqlDelight {
+  addRecordSource(
+    name = "hockeyPlayerOrigin",
+    database = "app.cash.backfila.client.sqldelight.hockeydata.HockeyDataDatabase",
+    tableName = "hockeyPlayer",
```

Comment:
Where is this set? This string doesn't seem to be in the sqldelight block above, is it auto-generated?

Reply:
I'm not sure I follow. The table name comes from the schema files. It does work with the SQLDelight plugin as well. This string comes from the `CREATE TABLE hockeyPlayer (` below.